src/plugins/GUITestBase/src/tests/common_scenarios/msa_editor/GTTestsMsaEditor.cpp

```diff
@@ -4615,6 +4615,32 @@ GUI_TEST_CLASS_DEFINITION(test_0090) {
     }
 }
 
+GUI_TEST_CLASS_DEFINITION(test_0091) {
+    // 1. Open file _common_data\scenarios\msa\nucl_with_leading_gaps.aln
+    GTFileDialog::openFile(os, testDir + "_common_data/scenarios/msa/nucl_with_leading_gaps.aln");
+    GTUtilsMsaEditor::checkMsaEditorWindowIsActive(os);
+    GTUtils::checkExportServiceIsEnabled(os);
+
+    // 2. Do document context menu {Export->Export aligniment to amino format}
+    // 3. Translate with "Include gaps"
+    GTUtilsDialog::waitForDialog(os, new ExportMSA2MSADialogFiller(os, -1, sandBoxDir + "GUITest_common_scenarios_msa_editor_test_0091.aln", true));
+    GTUtilsDialog::waitForDialog(os, new PopupChooser(os, QStringList() << MSAE_MENU_EXPORT << "amino_translation_of_alignment_rows"));
+    GTWidget::click(os, GTUtilsMdi::activeWindow(os), Qt::RightButton);
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    // 4. Copy to clipboard
+    GTUtilsMSAEditorSequenceArea::selectArea(os);
```

Comment:
nit: this normally should be a 1-liner, like getWholeData()..

Reply:
Sorry, don't get the idea

Reply:
If you want to check the sequence area content you can use a single line to access it: getWholeData. No need to copy/check clipboard for that. That will keep the test code minimal and focused in the key subject.
This is not a blocker anyway - only a suggestion.

Reply:
I took a note, but didn't fix it this time